### PR TITLE
feat: global mod bypass

### PR DESCRIPTION
**Pull request by tzarebczan**
see commit message

### CODE REVIEW

File: pkg/iapi/responses.go [hunk 1]
```diff
@@ -44,6 +44,43 @@ type CustomerListResponse struct {
 	} `json:"data"`
 }
 
+type UserMeResponse struct {
+	Success bool        `json:"success"`
+	Error   interface{} `json:"error"`
+	Data    struct {
+		Id                  int         `json:"id"`
```
Comment (anbsky) [inline pkg/iapi/responses.go:51]: Initialisms must be capitalized if possible: `ID`, `ExperimentalUI`

File: pkg/iapi/responses.go [hunk 2]
```diff
@@ -44,6 +44,43 @@ type CustomerListResponse struct {
...
+		Language            string      `json:"language"`
+		GivenName           interface{} `json:"given_name"`
```
[the remark this answers is not in the document]
Comment (tzarebczan) [reply pkg/iapi/responses.go:53]: shouldn't have trusted the copy/paste, chatgpt doing a better job.

**Comment by tzarebczan** [comment]
Gave it another shot, should be cleaner now. There's probably a way to even clean it up more, but should be fine.

File: app/query/processors.go [hunk 3]
```diff
@@ -568,3 +573,26 @@ func getStatusResponse(_ *Caller, ctx context.Context) (*jsonrpc.RPCResponse, er
 	rpcResponse.Result = response
 	return rpcResponse, nil
 }
+
+// Check and weakly return if the user is a mod, if errors occur false is assumed
```
Comment (anbsky) [inline app/query/processors.go:577]: A func’s doc comment should start with the function name.

**Review by anbsky — COMMENTED** [review]
Would've preferred at least a test for user/me response as a canary for unexpected internal-api format changes but otherwise looks good.

**Comment by tzarebczan** [comment]
Please ship when you're up if you can. We can add a test later, but the api should not change much.

**Pull request merged**